### PR TITLE
[chore] - Increase pagination limit

### PR DESCRIPTION
**Pull request by ahrav**
### Description:
- Some cleanup.
- Set pagination limit to the max value allowed (100).

### Checklist:
* [ ] Tests passing (`make test-community`)?
* [ ] Lint passing (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

File: pkg/sources/gitlab/gitlab.go [hunk 1]
```diff
@@ -293,45 +293,62 @@ func (s *Source) basicAuthSuccessful(apiClient *gitlab.Client) bool {
...
+		return nil, fmt.Errorf("unable to authenticate using %s, %w", s.authMethod, err)
+	}
+
+	uniqueProjects := make(map[int]*gitlab.Project)
+	var (
+		projects              []*gitlab.Project
+		projectsWithNamespace []string
+	)
+
+	// Used to filter out duplicate projects.
```
Comment (rosecodym) [inline pkg/sources/gitlab/gitlab.go:306]: do we see these? (i guess we could if the project list is updated during the operation)

**Review by rosecodym — APPROVED** [review]
thanks for looking at this! i still think there was some other cause of the slowdown though - 10m isn't 80% too long, it's, like, 99.9% too long

File: pkg/sources/gitlab/gitlab.go (hunk 1 above)
Comment (ahrav) [reply pkg/sources/gitlab/gitlab.go:306]: Yea, i was initially thinking the same thing, but i think the same project can appear under the user's list as well as the group's list.

**Comment by ahrav** [comment]
> thanks for looking at this! i still think there was some other cause of the slowdown though - 10m isn't 80% too long, it's, like, 99.9% too long

Yea there has to be something else going on here, this is by no means "THE FIX".

**Pull request merged**